axlearn/common/ssm_test.py

```diff
@@ -509,6 +511,66 @@ def test_prefill_states(self, dtype: jnp.dtype):
 
         assert_allclose(decoder_output, forward_outputs.data, atol=1e-6)
 
+    @parameterized.parameters(jnp.float32, jnp.bfloat16)
```

Comment:
Thanks for the tests!

Reply:
You're welcome! I'm glad I could be helpful.